sdk/python/feast/infra/feature_servers/gcp_cloudrun/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM python:3.11-slim
```

Comment:
why needed new docker-file why not using existing one.

Reply:
+1 , do we need a separate image?